backend/service/files.service.js

```diff
@@ -0,0 +1,40 @@
+const File = require('../models/files.model')
+
+const getFile = async (id) => {
+    return File.findById(id)
+}
+
+const getAllFilesByReference = async (reference) => {
+    return File.find({ referenceItem: reference })
```

Comment:
can we rename "reference" to ticketId to be explicit about what is being referenced

Reply:
This and subsequent comments resolved via call. Naming it reference as we wish to remain generic, and things might be tied to non-ticket concepts such as users